common/source/docs/common-paramosd.rst

```diff
@@ -0,0 +1,134 @@
+.. _common-paramosd:
+
+========================
+OSD based parameter menu
+========================
+
+.. image:: ../../../images/paramscreen.jpg
+    :target: ../_images/paramscreen.jpg
+    :width: 450px
+
+This allows to set and adjust paramaters using the ardupilot maxchip-based onboard OSD and an RC transmitter's stick input. 
+There's two additional OSD screens available (OSD5 and OSD6) that can hold up to 9 parameters each. Parameter values can be 
+adjusted in disarmed state only. The RC transmitter's roll and pitch stick input is used to navigate through the screen, pick 
+a parameter and adjust the parameter value.
+
```

Comment:
I think a few words about the concept of slots...that each screen has 9 slots, preselected via MAVProxy setup or default, but that any one can be changed directly from the screen using stick inputs....the param in each slot can be adjusted for it value also by stick input....sure you can figure out some wording ....

Reply:
fixed.
i think the defaults andy chose are copter-only params (ATC_RAT_...). might make sense to add defaults for plane too.

Reply:
Yes, you are right. Tell me what makes sense  here. I think also we should have some decent defaults for OSD6 that are different to OSD5

Reply:
perhaps one for plane and one for copter?